Content.Shared/Camera/GetEyeOffsetEvent.cs

```diff
@@ -0,0 +1,6 @@
+using System.Numerics;
+
+namespace Content.Shared.Camera;
+
+[ByRefEvent]
+public record struct GetEyeOffsetEvent(Vector2 Offset);
```

Comment:
Comment please.

Reply:
Done in [`b9b9ac5` (#29146)](https://github.com/space-wizards/space-station-14/pull/29146/commits/b9b9ac5417a70a92315c18c4c416b392ff4d5a83)